lib/identity_config.rb

```diff
@@ -95,6 +95,11 @@ def self.build_store(config_map)
     config.add(:aamva_verification_request_timeout, type: :float)
     config.add(:aamva_verification_url)
     config.add(:all_redirect_uris_cache_duration_minutes, type: :integer)
+<<<<<<< HEAD
+=======
+    config.add(:allow_otp_countdown_expired_redirect, type: :boolean)
+    config.add(:allowed_ialmax_providers, type: :json)
+>>>>>>> 514fb7d9e (Changelog: Internal, IALMAX, Create allowlise of service providers that can utilize the IALMax flow)
```

Comment:
got some git merge noise in here we should clean up
```suggestion
    config.add(:allow_otp_countdown_expired_redirect, type: :boolean)
    config.add(:allowed_ialmax_providers, type: :json)
```

Reply:
yes! sorry the save didn't take in my editor (i am having a day)